Decred Wallet/Features/Settings/SettingsController.swift

```diff
@@ -245,12 +245,10 @@ class SettingsController: UITableViewController  {
         UserDefaults.standard.removePersistentDomain(forName: Bundle.main.bundleIdentifier!)
         UserDefaults.standard.synchronize()
         
-        UIApplication.shared.keyWindow?.rootViewController?.dismiss(animated: false, completion: nil)
-        
-        if let appDelegate = UIApplication.shared.delegate as? AppDelegate {
-            let startScreen = Storyboards.Main.initialViewController()
-            appDelegate.setAndDisplayRootViewController(startScreen!)
-        }
+        // Stop calling wallet.bestBlockTimestamp() to update the best block age displayed on nav menu.
+        self.navigationMenuViewController()?.stopRefreshingBestBlockAge()
+        let startScreen = Storyboards.Main.initialViewController()
```

Comment:
Empty line space before line 250 please. Lines 248-249 are not related to lines 250-251.